apps/site/pages/en/about/get-involved/index.md

```diff
@@ -5,32 +5,30 @@ layout: about
 
 # Get Involved
 
-If you are interested in getting involved with the Node.js community, there are many ways to do so. The Node.js project is a large and diverse community, and there are many ways to contribute, beyond just writing code.
+If you are interested in getting involved with the Node.js community, there are many ways to do so. The Node.js project is a large and diverse community with many ways to contribute beyond just writing code.
 
 ## Community Discussion
 
-- The [Node.js official Discord server](/discord) is a place to chat with other Node.js developers and getting official news from the Node.js project.
-- The [GitHub issues list](https://github.com/nodejs/node/issues) is the place for discussion of Node.js core features and if you have questions about Node.js, you can use the [github discussions](https://github.com/orgs/nodejs/discussions).
-- The [`nodejs/help`](https://github.com/nodejs/help/issues) repository is the place to ask questions about Node.js.
-- The official Node.js Twitter account is [nodejs](https://twitter.com/nodejs).
```

Comment:
Is there a reason why this resource was removed?

Reply:
Twitter is not a Node.js communication platform. You can post things with @node, but it is not really a place to talk about Node. It is more like a social network like the others mentioned below, such as Bluesky and Linkedin.